lib/rollbar/configuration.rb

```diff
@@ -143,7 +212,8 @@ def merge!(options)
     def use_active_job(options = {})
       require 'rollbar/delay/active_job'
 
-      Rollbar::Delay::ActiveJob.queue_as(options[:queue] || Rollbar::Delay::ActiveJob.default_queue_name)
+      Rollbar::Delay::ActiveJob.queue_as(options[:queue] ||
+      Rollbar::Delay::ActiveJob.default_queue_name)
```

Comment:
I think some indentation might be missing here

Reply:
Rubocop's default setting is 'aligned', which is what we see above. Rubocop also allows 'indented', which would need to be an override in the rubocop config, otherwise indenting will produce a violation.

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineOperationIndentation